CRT.c

```diff
@@ -194,27 +194,27 @@ static int CRT_colorSchemes[LAST_COLORSCHEME][LAST_COLORELEMENT] = {
       [CPU_SOFTIRQ] = ColorPair(Magenta, Black),
       [CPU_STEAL] = ColorPair(Cyan, Black),
       [CPU_GUEST] = ColorPair(Cyan, Black),
-      [PANEL_EDIT] = ColorPair(White, Blue),
-      [SCREENS_OTH_BORDER] = ColorPair(Blue, Blue),
-      [SCREENS_OTH_TEXT] = ColorPair(Black, Blue),
+      [PANEL_EDIT] = A_BOLD | ColorPair(White, Blue),
```

Comment:
Why did you change that? Seems unrelated?

Reply:
My change was too quick an too dirty. 😜

Reply:
Still in the latest iteration of the PR